api/v1alpha1/condition_consts.go

```diff
@@ -37,3 +37,23 @@ const (
 	// DeletedReason (Severity=Info) documents a condition not in Status=True because the underlying object was deleted.
 	DeletedReason = "Deleted"
 )
+
+// Conditions related to the VirtualMachineImages
+const(
+	// VirtualMachineImageOsTypeSupportedCondition denotes that the OS type in the virtual machine image object is
+	// supported by VMService. A VirtualMachineImageOsTypeSupportedCondition is marked true
+	// - If OS Type is of Linux Family
+	// - If OS Type is supported by hosts in the cluster
+	VirtualMachineImageOsTypeSupportedCondition ConditionType = "VirtualMachineImageOsTypeSupported"
+
+	// VirtualMachineImageSupportedCondition denotes image compatibility with VMService. VMService expects
+	// VirtualMachineImage to be prepared by VI admin for VMService consumption or is a photon image.
+	VirtualMachineImagePreparedForVmServiceCondition ConditionType = "VirtualMachineImagePreparedForVmServiceCondition"
+)
+
+// Condition.Reason for Conditions related to VirtualMachineImages
+const (
+	VirtualMachineImageOsTypeNotSupportedReason = "VirtualMachineImageOsTypeNotSupported"
```

Comment:
Here and below, maybe add a comment indicating what this means? This is self explanatory, but it would help with the documentation generation.